picard/script/functions.py

```diff
@@ -1348,3 +1348,22 @@ def func_unique(parser, multi, case_sensitive="", separator=MULTI_VALUED_JOINER)
     if not case_sensitive:
         multi_value._multi = list({v.lower(): v for v in multi_value}.values())
     return multi_value.separator.join(sorted(set(multi_value)))
+
+
+@script_function(eval_args=False, documentation=N_(
+    """`$cleanmulti(name,separator="; ")`
+
+Removes empty elements from the multi-value variable `name`.
+
+Example:
+
+    $cleanmulti(%foo%)
+"""
+))
+def func_cleanmulti(parser, multi, separator=MULTI_VALUED_JOINER):
```

Comment:
Not sure about the function name, but I didn't find better.

Reply:
I agree that there's probably a better name, but I can't think of one right now.  I'll happily change it if someone comes up with a better suggestion.  The only other one I'm thinking right now is `$removeemptymulti()` but I'm not sure that's any better.

Reply:
Maybe `$delemptymulti()`?

Reply:
Not convinced by `$delemptymulti()` either. @phw @Sophist-UK @mineo any idea?